functions-and-scope.js

```diff
@@ -15,9 +15,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 6
-for (let i = 0; i < 8; i++){
+console.log("========== opdracht 1A ==========")
+studentsGraduated = 0;
 
+for(let i = 0; i < grades.length; i++){
+    if (grades[i] >= 8){
+        studentsGraduated = studentsGraduated +1;
```

Comment:
Dit kun je versimpelen:
```suggestion
        studentsGraduated++;
```